### PR TITLE
ft(basic):change header properties to match basics properties from js…

### PR DESCRIPTION
**Pull request by Mr-emeka**
…on resume data
Please kindly crosscheck. @aamay001

### CODE REVIEW

File: src/components/Resume/Header.js [hunk 1]
```diff
@@ -25,71 +25,95 @@ export const Header = ({
...
-    {showAddress && (
-      <ul data-testid="Address">
-        <li>{header.address}</li>
-        <li>{header.city}</li>
-        <li>{header.state}</li>
-        <li>{header.zip}</li>
-        <li>{header.country}</li>
+    <header className="resume-header" style={{ fontFamily: font }}>
+      <div>
+        <img src={header.picture} alt={`${header.name}-picture`} style={{ borderRadius: '50%', width: 'auto', objectFit: 'cover', height: '70px' }} />
```
Comment (aamay001) [inline src/components/Resume/Header.js:30]: I am going to add some clarification to the instructions. The header image is good but, I want to include the ability to toggle it on and off. Additionally, I am going to add a helper method to migrate the old schema to the new schema.

File: src/resume-data.js [hunk 2]
```diff
@@ -1,16 +1,30 @@
 const Resume = {
   header: {
```
Comment (aamay001) [inline src/resume-data.js:2]: `header` object should be renamed to `basics`.

**Review by aamay001 — CHANGES_REQUESTED** [review]
I added some comments to the changed files. I am going to close this PR. Please PR into the development branch.

**Comment by aamay001** [comment]
> I added some comments to the changed files. I am going to close this PR. Please PR into the development branch.

Actually, I just pointed the PR to the correct branch. It is staying open.

File: src/resume-data.js (hunk 2 above)
Comment (Mr-emeka) [reply src/resume-data.js:2]: okay.

File: src/components/Resume/Header.js (hunk 1 above)
Comment (Mr-emeka) [reply src/components/Resume/Header.js:30]: okay, I would add the ability to toggle it on and off. Also, the helper method to migrate the old schema I think you lost me right there.

**Comment by aamay001** [comment]
@Mr-emeka Additional clarifications and instructions have been added to #72. Please review and resubmit. Closing.

**Pull request closed without merging**